### ISSUE TITLE
Sync always stuck on 16% 

### ISSUE THREAD

**Issue by mernmx** [croneter/PlexKodiConnect#1063]
am trying to sync my plex share for the past two days and it always get stuck at 16% for tv shows section which is 16999 episode the exact number every time i have tried with nvidia shield and pc with different version of pkc same result all day long

this is my log https://gofile.io/?c=yyq8U7

**Comment by croneter** [comment]
The log does not show anything. Could you please share your pms with me?

**Comment by mernmx** [comment]
I wish i could do thath but Its a paid pms so i can't but i can give you my account so you can check if that's ok with you

**Comment by croneter** [comment]
I don't really want to do that. Let's wait for https://github.com/croneter/PlexKodiConnect/issues/1045

**Comment by mernmx** [comment]
Ok i will invite add you to the server can you please give ur username cause he have a 2 trial i think its enough

**Comment by croneter** [comment]
Croneter :wink:

**Comment by mernmx** [comment]
Done Sent it 
Try to sync tv shows folder and see what happens

**Comment by croneter** [comment]
Sorry, I did not yet receive an invite for the Plex user `Croneter`. You sure you sent one?

**Comment by mernmx** [comment]
Done i have resend it

**Comment by croneter** [comment]
Got it, thanks!

**Comment by croneter** [comment]
OK, I can replicate the bug. It get's stuck here:
![image](https://user-images.githubusercontent.com/4637940/68960188-f3e95000-07cf-11ea-9e9b-1c45e21986e3.png)
```
2019-11-15 17:45:16.131 T:12824  NOTICE: PLEX.tvshows: ADD episode plex_id: 31018 - A Tough Prospect
2019-11-15 17:45:16.210 T:12824   DEBUG: PLEX.sync.full_sync: Start or continue processing section TV Shows (episodes)
```
Last one hangs. Will look into it

**Comment by mernmx** [comment]
Yes that's the same issue 
Waiting for ur solution and thank you for your efforts

**Comment by mernmx** [comment]
any updates?

**Comment by croneter** [comment]
I know what's wrong. But I need to tinker with the code extensively... 😞

**Comment by mernmx** [comment]
I wish i can help you cause i really need it plex sucks without your magic

**Comment by mernmx** [comment]
Any news?

**Comment by croneter** [comment]
See the Pull Request linked just above your last post. You can follow the progress there...

**Comment by croneter** [comment]
Does PKC BETA version 2.10.5 solve your issue?

**Comment by mernmx** [comment]
I have installed the new beta but after the initial setup it gives me an error 

this is the log file:[https://xfl.jp/XEnoUs](url)

**Comment by mernmx** [comment]
https://xfl.jp/XEnoUs

**Comment by croneter** [comment]
Thanks for that! Please try beta version 2.10.6, online in a few seconds

**Comment by croneter** [comment]
Fixed?

**Comment by mernmx** [comment]
this is great everything seems to work now.

i wish you the best and thank you for your efforts

**Comment by croneter** [comment]
Nice!! Thanks for letting me know

**Comment by croneter** [comment]
This should be fixed for everyone with version 2.10.12
